### PR TITLE
Andrii sas

### PR DESCRIPTION
**Pull request by Andriisas**

### CODE REVIEW

**Review by Oleg-Smal-git — APPROVED** [review]
LGTM.

File: Tasks/task_325.py [hunk 1]
```diff
@@ -10,14 +14,15 @@ def simle_value(num):
...
+    list_values = []
+    for x in range(1, n + 1):
+        if n % x == 0 and prime_number(x) == True:
+            list_values.append(x)
 
-        return list_values
+    return list_values
 
 
 task_325 = Task(
```
Comment (CheshirLvova) [inline Tasks/task_325.py:28]: change this part like this:

task_325 = Task(
    name="task_325",
    description=
    """
    Please, add your description
    """,
    body=_task,
    test="Tasks/test_task_325.py"
)

File: Tasks/test_task_328.py [hunk 2]
```diff
@@ -1,17 +1,11 @@
...
             181, 191, 193, 197, 199, 211, 223, 227, 229, 233, 239, 241, 251, 257, 263, 269, 271, 277,
             281, 283, 293, 307, 311, 313, 317, 331, 337, 347, 349, 353, 359, 367, 373, 379, 383, 389,
             397, 401, 409, 419, 421, 431, 433, 439, 443, 449, 457, 461, 463, 467, 479, 487, 491, 499,
             503, 509, 521, 523, 541
-        ])
-
-if __name__ == '__main__':
-    unittest.main()
\ No newline at end of file
+        ]
```
Comment (CheshirLvova) [inline Tasks/test_task_328.py]: add newline

File: Tasks/test_task_328.py [hunk 3]
```diff
@@ -1,17 +1,11 @@
-import unittest
 from Tasks.task_328 import task_328
 
-class Test328(unittest.TestCase):
-    def test_328(self):
-        result = task_328._task()
-        self.assertEqual(result, [
+def test_result():
+    assert task_328._task() == [
```
Comment (CheshirLvova) [inline Tasks/test_task_328.py]: there must be ...task_328.run_body()...

**Review by CheshirLvova — CHANGES_REQUESTED** [review]

**Review by CheshirLvova — APPROVED** [review]
nice)

File: Tasks/task_086v.py [hunk 4]
```diff
@@ -1,23 +1,19 @@
...
-    if value > 0:
-        string_value = str(value)
-        list_values = []
-        for x in string_value:
-            list_values.append(int(x))
-        return list_values[0]
-    else:
-        return 'Enter Natural number!'
-        
+def _task(value : int) -> int:
```
Comment (v-lavrushko) [inline Tasks/task_086v.py:5]: please add docstring

File: Tasks/test_task_113a.py [hunk 5]
```diff
@@ -1,7 +1,6 @@
 import unittest
 from Tasks.task_113a import task_113a
 
-
```
Comment (v-lavrushko) [inline Tasks/test_task_113a.py]: every class - 2 blank lines (PEP8)

File: Tasks/task_325.py [hunk 6]
```diff
@@ -1,6 +1,10 @@
 from task import Task
+from typing import List
+from task_error import TaskError
 
-def simle_value(num):
+def prime_number(num : int) -> bool:
```
Comment (v-lavrushko) [inline Tasks/task_325.py:5]: there is a better algorithm. Can help with it

**Review by v-lavrushko — CHANGES_REQUESTED** [review]

**Review by OlehHnyp — APPROVED** [review]

**Review by v-lavrushko — APPROVED** [review]

**Pull request merged**